.gitmodules

```diff
@@ -18,9 +18,9 @@
   path = CMakeModules
   url = https://github.com/NOAA-EMC/CMakeModules
   branch = develop
-[submodule "CMEPS"]
+[submodule "CMEPS-interface/CMEPS"]
```

Comment:
Why do we need this change?

Reply:
We don't. Not sure why it was introduced.

Reply:
I think the gocart-interface change can still be made without affecting the discussion around ecbuild/jedi/... stuff.

Reply:
Commit 1d09a64e0cfabcc57c807a615d415dd64a142bfc reverted all unnecessary changes in `.gitmodules`